### PR TITLE
SPE-58 Migrate Dex Prices & Update Airdrop Claim Models - Ready for review

### PR DESCRIPTION
**Pull request by henrystats**
# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.

### CODE REVIEW

**Comment by couralex6** [comment]
Feel free to add these in your dex_prices pr so you can test all the models in the CI

**Comment by henrystats** [comment]
okeee ser, thank you @couralex6

**Comment by henrystats** [comment]
@couralex6 I added dex prices

**Review by couralex6 — APPROVED** [review]
lgtm. thanks!

File: models/gearbox/ethereum/gearbox_ethereum_airdrop_claims.sql [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {{
     config(
+        tags = ['dunesql', 'static'],
         schema = 'gearbox_ethereum',
         alias = alias('airdrop_claims'),
-        materialized = 'incremental',
+        materialized = 'table',
```
Comment (jeff-dude) [inline models/gearbox/ethereum/gearbox_ethereum_airdrop_claims.sql:6]: thank you for this enhancement to a static table 🙏

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**